ecc/cmd/main.go

```diff
@@ -15,6 +15,11 @@ import (
 var logger = flogging.MustGetLogger("ecc")
 
 func main() {
+
+	// we can control logging via FABRIC_LOGGING_SPEC, the default is FABRIC_LOGGING_SPEC=INFO
+	// For more fine grained logging we could also use different log level for loggers.
+	// For example: FABRIC_LOGGING_SPEC=ecc=DEBUG:ecc_enclave=ERROR
```

Comment:
Why not just check whether FABRIC_LOGGIN_SPEC exists as environment variable and take it from there?  This should work both with current container approach as well with external-builder. in the latter case, of course we would have more leeway, e.g., via command options like mock server, but environment variable works in both cases and is also how fabric itself handles it.   Insofar, we might even change mockserver to also use environment variable, at least in addition? (Note that this gives you also more control over logging in case you need it via a more complicated logging spect ...

Reply:
flogger already takes care of reading `FABRIC_LOGGIN_SPEC` from env. There is no need for additional code here. See [this](https://github.com/hyperledger/fabric/blob/master/common/flogging/logging.go#L87).

That is, the loggers used in mock server and ecc will read whatever is specified in `FABRIC_LOGGIN_SPEC`.

- Log levels are defined [here](https://github.com/hyperledger/fabric/blob/master/common/flogging/levels.go#L36)

Reply:
Oops, i could have sworn i saw here a commented out flogging.ActivateSpec.  I guess too early in the morning.  In any case, even better. Then we should be good to go for (controlled) logging in chaincodes, see my comment on peer forwarding environment variables ...